rolling/distribution.yaml

```diff
@@ -51,6 +51,15 @@ repositories:
       version: ros2
     status: maintained
   actuator_msgs:
+    doc:
+      type: git
+      url: https://github.com/rudislabs/actuator_msgs.git
+      version: 0.0.1
```

Comment:
This will usually be a branch rather than a tag, since it's reasonable to want to update your documentation without a complete release. That of course relies on your specified branch being compatible with the current release. This isn't ab blocker though just an observation and can be changed in a subsequent manual PR.

Reply:
Ah, good to know for the future, thanks.